app/pom.xml

```diff
@@ -70,7 +71,19 @@
             <artifactId>lombok</artifactId>
             <scope>compile</scope>
         </dependency>
-        
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+        </dependency>
```

Comment:
What did you need in `commons-io`?

Reply:
Ah that was before I knew about the registry IoUtil. I will remove now